src/test/resources/services-local.conf

```diff
@@ -14,23 +14,22 @@
 
 # This file is read when runLocal = true in application.conf (default)
 
-services {
-    auth-login-stub {
-      protocol = http
-      host = localhost
-      port = 9949
-    }
-    address-lookup-frontend {
-      protocol = http
-      host = localhost
-      port = 9028
-      postcode = "ZZ9Z 9TT"
-    }
-    digital-disclosure {
-      host = localhost
-      port = 15003
-    }
+# For local development convenience, host and protocol will have the following defaults:
+#      protocol = http
+#      host = localhost
 
+# Example 1: (the base url for helloworld-service is https://127.0.0.1:9000)
+# helloworld-service {
+#      protocol = https
+#      host = 127.0.0.1
+#      port = 9000
+# }
 
+# Example 2: (the base url for helloworld-service is http://localhost:9000)
+# helloworld-service.port = 9000
 
+services {
```

Comment:
There is no need to define `protocol` and `host` when defining local services as they are defined by default (as per the comment in the template). Setting `service-name.port = 1234` will suffice.